scripts/clone_template.py

```diff
@@ -86,7 +88,6 @@ def main(**kwargs):
     if kwargs.get('deploy'):
         kwargs['configure'] = True
         kwargs['outfile'] = 'appliance_ip_address_1'
-        provider_data = cfme.utils.conf.provider_data
```

Comment:
This is covered by the direct import now, which without the --deploy option is never accessed and runs fine without the corresponding file.